styles/main.css

```diff
@@ -0,0 +1,57 @@
+/* UTILS */
+
+@import url("utils/container.css");
+@import url("utils/flex.css");
+@import url("utils/grids.css");
+@import url("utils/fonts.css");
+@import url("utils/lists.css");
+@import url("utils/button.css");
+@import url("utils/utilities.css");
+@import url("utils/gutter.css");
+@import url("utils/width.css");
+
+/* COMPONENTS */
+
+@import url("components-styles/avatar.css");
+@import url("components-styles/alert.css");
+
+:root {
+  --primary-color-xxlight: #f1f0ff;
```

Comment:
To name variables, use this notation `--COLOR-PRIMARY-XXLIGHT`. it will help to notice that it is a variable.

Reply:
And UI looks amazing, Nice work 👍

Reply:
Thanks a lot for reviewing! I've made the following changes for naming CSS variables.